motion/p5js/connected-elements/index.md

```diff
@@ -0,0 +1,9 @@
+---  
+layout: p5js
+author: Marius Claßen
+author-url: https://github.com/mariusclassen
+title: Air Dancer
+tags: [P5.js, motion, law of connected elements]
+permalink: https://github.com/fabianmoronzirfas/gestalten-in-code/tree/master/motion/air-dancer
```

Comment:
the permalink should only be 

/motion/-air-dancer/

You also need to rename the folder your code is in.